eclass/font.eclass

```diff
@@ -46,21 +46,17 @@ FONTDIR=${FONTDIR:-/usr/share/fonts/${FONT_PN}}
 # Array containing fontconfig conf files to install.
 FONT_CONF=( "" )
```

Comment:
Remove the commit to `font.eclass`, will get all cleaned up by https://github.com/gentoo/gentoo/pull/20027